doc/life_of_a_language_feature.md

```diff
@@ -1,116 +1,141 @@
-# Dart language evolution process
+# Lifecycle of a Dart Language Feature
```

Comment:
Further nit: I'd either rename the file or change this to "Lifecycle of a Language Feature", probably the latter given that Dart should be obvious. We ain't designin' Perl features here. :)

Reply:
Fair enough haha. Removed the Dart part.